PluginsAndFeatures/azure-toolkit-for-intellij/rider/build.gradle

```diff
@@ -43,7 +43,17 @@ intellij {
 
     // Workaround for https://youtrack.jetbrains.com/issue/IDEA-179607
     def extraPlugins = [ "rider-plugins-appender" ]
-    plugins = ['DatabaseTools', 'JavaScriptLanguage', 'terminal', 'restClient'] + extraPlugins
+    plugins = ['DatabaseTools', 'JavaScriptLanguage', 'terminal', 'restClient', "com.jetbrains.rider.fsharp:$rider_fsharp_plugin_version"] + extraPlugins
+
+    // HACK: -- Load F# plugin from custom repository (our local disk)
```

Comment:
@mfilippov @auduchinok Same thing here, manually referencing the F# plugin on frontend from a custom disk-based URL.

Reply:
To fix it we need to add two-phase build for bundled plugins. Or auto-upload to the gallery.

Reply:
Auto-upload seems like an option, a least on the consuming side.

Reply:
Makes sense to upload EAP builds to the gallery - I want to do the same with the Unity ReSharper plugin (not Rider). Nightly seems overkill - could we upload those to the snapshots repository?